istantest.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
```

Comment:
It's a [Shebang](http://en.wikipedia.org/wiki/Shebang_%28Unix%29) interpreter directive.